wordle-web/components/PrerollAd.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-card width="800" height="800">
+    <v-container class="overflow-hidden" fluid>
+      <v-row justify="center">
+        <v-col>
+          <!-- <Adsense
+            data-ad-client="ca-pub-1431122584842002"
+            data-ad-slot="9479195735"
+          ></Adsense> -->
+          <!-- That doesn't work without a top-level domain, anyway. Here's a fake ad instead -->
+          <a href="https://youtu.be/dQw4w9WgXcQ" target="_blank">
```

Comment:
So I sent this link to Grant, and he immediately calls me dying of laughter. This was very, very funny. Awesome job guys!